### DISCUSSION TITLE
cannot be displayed in tags view and Solution

### DISCUSSION THREAD

**Discussion by sci268** [marcusolsson/obsidian-projects#422] [Help]
Provide a small problem, which may not be a bug
Solutions that cannot be displayed in tags view

Expect: It can be improved to that individual tags errors will not affect other view display, that is, increase the compatibility of tags views. thanks！

Q：When a tag is incorrectly filled in the library, all tag views become invalid.
Even if the label library of other labels is used
![image](https://user-images.githubusercontent.com/76660932/217031892-3893b43b-4303-42b9-a057-8a665cd965a1.png)

S：when changed ，all became ok
![image](https://user-images.githubusercontent.com/76660932/217032490-19e841a0-0e8a-4537-afb3-c676c212fa49.png)

**Comment by marcusolsson** [comment]
Closing in favor of #414.
